### PR TITLE
update Community section and other fixes

### PR DESCRIPTION
**Pull request by dcalvin**
- Added Community section
- Added benchmark link
- Other wording or consistency fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ You can try Chaos Mesh on your local K8s environment deployed using `kind` or `m
    kubectl -n kube-system get pods -l app=helm
    ```
 
-4. Install `chaos-operator` as suggested in [Deploy Chaos Mesh](#deploy-chaos-mesh).
+4. Install `chaos-mesh` on `minikube` kubernetes cluster as suggested in [Install Chaos Mesh](#install-chaos-mesh).
```
Comment (yikeke) [inline README.md]: Suggest keeping the expressions of this line consistent with that of Line 114~

File: README.md [hunk 2]
```diff
@@ -169,14 +169,14 @@ metadata:
   namespace: chaos-testing
 spec:
   action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure
-  mode: one # the mode to run chaos action; supported mode are one/all/fixed/fixed-percent/random-max-percent
+  mode: one # the mode to run chaos action; supported modes are one/all/fixed/fixed-percent/random-max-percent
   duration: "60s" # duration for the injected chaos experiment
   selector: # pods where to inject chaos actions
     namespaces:
       - tidb-cluster-demo
```
Comment (yikeke) [inline README.md]:
```suggestion
      - tidb-cluster-demo  # the namespace of the TiDB cluster you've deployed 
```

File: README.md [hunk 3]
```diff
@@ -204,13 +204,13 @@ kubectl apply -f pod-failure-example.yaml
...
 
-#### Warch your chaos experiments in Dashboard
+#### Watch your chaos experiments in Dashboard
 
 Chaos Dashboard is currently only available for TiDB clusters. Stay tuned for more supports or join us in making it happen.
 
 > **Note:**
 >
-> Make sure you have used the [option](#deploy-chaos-mesh) to deploy Chaos Mesh with Chaos Dashboard. If Chaos Dashboard wasn't installed in your Chaos Mesh, you need to install it by upgrading Chaos Mesh:
+> If Chaos Dashboard wasn't installed in your Chaos Mesh in your earlier, you need to install it by upgrading Chaos Mesh:
```
Comment (yikeke) [inline README.md]:
```suggestion
> If Chaos Dashboard is not installed when you install Chaos Mesh earlier, you need to install it by upgrading Chaos Mesh:
```

File: README.md [hunk 4]
```diff
@@ -222,6 +222,13 @@ kubectl port-forward -n chaos-testing svc/chaos-dashboard 8080:80
 
 Then you can access [`http://localhost:8080`](http://localhost:8080) in browser.
 
+## Community
+
+Please reach out for bugs, feature requests, and other issues via:
+
+- The #chaos-mesh channel in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```
Comment (yikeke) [inline README.md]:
```suggestion
- The #chaos-mesh channel in the [TiDB Community](https://pingcap.com/tidbslack) slack workspace.
```

File: README.md [hunk 5]
```diff
@@ -159,7 +159,7 @@ You can follow the instructions in the following two documents to deploy a TiDB
 
 ### Define chaos experiment config file
 
-In this sample experiment config file, we will define a chaos experiment to kill one tikv pod randomly:
+The chaos experiement configuration is defined in a .yaml file. The following sample file (`pod-kill-example.yaml`) defines a chaos experiment to kill one tikv pod randomly:
```
Comment (mahjonp) [inline README.md]:
```suggestion
The chaos experiement configuration is defined in a .yaml file. The following sample file (`pod-kill-example.yaml`) defines a chaos experiment to kill one tikv pod randomly every 60 seconds:
```

File: README.md [hunk 6]
```diff
@@ -111,14 +111,14 @@ You can try Chaos Mesh on your local K8s environment deployed using `kind` or `m
    kubectl cluster-info
    ```
 
-5. Install `chaos-mesh` on `kind` kubernetes cluster as suggested in [Deploy Chaos Mesh](#deploy-chaos-mesh).
+5. Install `chaos-mesh` on `kind` kubernetes cluster as suggested in [Install Chaos Mesh](#install-chaos-mesh).
 
 #### Deploy with `minikube`
 
 1. Start a `minikube` kubernetes cluster. Make sure you have installed [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), and run the script below to start a local `minikube` cluster
```
Comment (dcalvin) [inline README.md]:
```suggestion
1. Start a `minikube` kubernetes cluster. Make sure you have installed [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
```

File: README.md [hunk 7]
```diff
@@ -169,14 +169,14 @@ metadata:
   namespace: chaos-testing
 spec:
   action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure
-  mode: one # the mode to run chaos action; supported mode are one/all/fixed/fixed-percent/random-max-percent
+  mode: one # the mode to run chaos action; supported modes are one/all/fixed/fixed-percent/random-max-percent
   duration: "60s" # duration for the injected chaos experiment
   selector: # pods where to inject chaos actions
     namespaces:
-      - tidb-cluster-demo
+      - tidb-cluster-demo  # the namespace of the TiDB cluster you've deployed
```
Comment (dcalvin) [inline README.md]:
```suggestion
      - tidb-cluster-demo  # the namespace of the system under test (SUT) you've deployed 
```

File: README.md [hunk 8]
```diff
@@ -169,14 +169,14 @@ metadata:
...
   action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure
-  mode: one # the mode to run chaos action; supported mode are one/all/fixed/fixed-percent/random-max-percent
+  mode: one # the mode to run chaos action; supported modes are one/all/fixed/fixed-percent/random-max-percent
   duration: "60s" # duration for the injected chaos experiment
   selector: # pods where to inject chaos actions
     namespaces:
-      - tidb-cluster-demo
+      - tidb-cluster-demo  # the namespace of the system under test (SUT) you've deployed 
     labelSelectors:
       "app.kubernetes.io/component": "tikv"
```
Comment (dcalvin) [inline README.md]:
```suggestion
      "app.kubernetes.io/component": "tikv" # the label of the pod for chaos injection
```

**Pull request merged**